### DISCUSSION TITLE
Feedback on benefits

### DISCUSSION THREAD

**Discussion by Axelgoris99** [community/community#59478] [GitHub Education]
### Hi everyone,

Heeeeej guys 👋 

First of all, thanks for this amazing opportunity. Can't wait to get started!
I'd like to bring some feedback on some of the stuff written, but feel free to tell me if I'm in the wrong place. It's not really bugs, more like small things that aren't perfectly clear. (And considering I'm at it, i'm gonna be a bit nitpicky... Sorry! 🙌 )

- [Datacamp](https://www.datacamp.com/pricing): "Free 3-month individual subscription for students". I think you're speaking about the "premium for individuals" is that correct?
- [Icons8](https://icons8.com/pricing): "3-month All Access subscription with icons, photos, illustrations, and music". The "All Access" in capitals make it seems like it exists while it is not a specific plan.
- [Termius](https://termius.com/pricing): "Free access to the Premium plan while you're a student". Same here, they have a pro plan, but no premium plan. 
- [Polypane](https://polypane.app/pricing/): "Polypane free for 1 year". They have an individual plan, i guess that's it but not sure. Maybe business? 😜 
- [GitKraken](https://www.gitkraken.com/git-client/pricing): "Free GitKraken Client Pro ($59 value) while you’re a student". It's actually $59 per year!
- [StreamYard](https://streamyard.com/pricing?planType=individuals):"Free access to the Essential Plan while you are a student". They don't have an essential plan. Maybe the basic one?
- [Replit](https://replit.com/pricing) "Six free months of private repls". Only that? or is it the "hacker" plan or something of the sort?
- [Typeform](url): "Free professional plan for 1 year". The actual reason I started writing that! They don't have a professional plan, it changed into "plus" plan in 2021. Or is it the basic one?
- [Freshpaint](https://www.freshpaint.io/pricing?): "Free Freshpaint Growth Plan while you are a student". I think it does not exist anymore? or does it?
- [Honeybadger](https://www.honeybadger.io/plans/): "Free Small account for 1 year". Can't find what is a small plan 😿 
- [Twilio](https://www.twilio.com/blog/twilio-perks-students-and-educators-now-available-github-education): not on the list?

I think I covered them all, feel free to tell me if I'm missing some (or if I'm being really picky/unfair!).
Sorry for the inconvenience!

Have a lovely morning, afternoon, evening, day or night! (🌃 for me)
Axel

Edit1: Twilio added

**Comment by steven88663** [comment]
twilio
